src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -138,7 +138,18 @@ private function parseImports($content, $file)
             }
 
             $this->setCurrentDir($defaultDirectory);
-            $this->import($import['resource'], isset($import['type']) ? $import['type'] : null, isset($import['ignore_errors']) ? (bool) $import['ignore_errors'] : false, $file);
+
+            if (!$this->isAbsolutePath($import['resource'])) {
+                $import['resource'] = $defaultDirectory.DIRECTORY_SEPARATOR.$import['resource'];
+            }
+
+            if (array() !== $files = glob($import['resource'], defined('GLOB_BRACE') ? GLOB_BRACE : 0)) {
+                foreach ($files as $resource) {
+                    $this->import($resource, isset($import['type']) ? $import['type'] : null, isset($import['ignore_errors']) ? (bool) $import['ignore_errors'] : false, $file);
+                }
+            } else {
+                $this->import($import['resource'], isset($import['type']) ? $import['type'] : null, isset($import['ignore_errors']) ? (bool) $import['ignore_errors'] : false, $file);
```

Comment:
I need to rethink this part a bit, because without this, you don't get the exception if a file doesn't exist